Lombiq.HelpfulLibraries.Samples/Manifest.cs

```diff
@@ -0,0 +1,10 @@
+using OrchardCore.Modules.Manifest;
+
+[assembly: Module(
+    Name = "Lombiq.HelpfulLibraries.Samples",

```

Comment:
A human-readable  version of this, see https://lombiq.atlassian.net/wiki/spaces/DEV/pages/19496967/Creating+a+new+Visual+Studio+project+or+solution.